lib/web/ui/webconfig.go

```diff
@@ -61,4 +61,6 @@ type WebConfigAuthSettings struct {
 	SecondFactor string `json:"second_factor,omitempty"`
 	// Providers contains a list of configured auth providers
 	Providers []WebConfigAuthProvider `json:"providers,omitempty"`
+	// LocalAuth enables/disables local logins
```

Comment:
// LocalAuth is a flag that enables local authentication

Reply:
revised